src/Pact/Types/Runtime.hs

```diff
@@ -181,9 +181,9 @@ makeLenses ''PactExec
 
 -- | Indicates level of db access offered in current Eval monad.
 data Purity =
-  -- | No database access at all.
+  -- | Only access to systables.
```

Comment:
This is a bit redundant, but we might want to include that this is read-only: "Pure is only allowed to read systables."

Reply:
Good call. Will change